docs/getting-started/self-healing-rollback.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[self-healing-rollback]]
+= Configure self-healing rollback for Windows endpoints
+
+{endpoint-cloud-sec}'s self-healing feature rolls back file changes and processes on Windows endpoints when a prevention alert is generated by enabled protection features. All activity on the host reverts to its state five minutes before the prevention alert.
+
+This can help contain the impact of malicious activity, as {endpoint-cloud-sec} not only stops the activity but also erases any attack artifacts deployed prior to detection.
+
+Self-healing rollback is a https://www.elastic.co/pricing[Platinum or Enterprise subscription] feature, and it's only supported for Windows endpoints.
+
+[CAUTION]
+====
+This feature can cause permanent data loss, since it overwrites recent changes and deletes recently added files on the host. Self-healing rollback targets the changes related to a detected threat, but it may also include incidental actions that aren't directly related to the threat.
```

Comment:
```suggestion
This feature can cause permanent data loss since it overwrites recent changes and deletes recently added files on the host. Self-healing rollback targets the changes related to a detected threat, but may also include incidental actions that aren't directly related to the threat.
```